R/tmb.R

```diff
@@ -595,15 +596,15 @@ fit_mmrm <- function(formula,
   args <- with(
     tmb_object,
     c(
-      list(par, fn, gr, control = control$optimizer_control),
-      control$optimizer_args
+      list(par, fn, gr),
+      attr(control$optimizers[[1]], "args")
```

Comment:
Here the position access feels a bit risky. Can we make this more robust?

Reply:
we can add an assertion to make sure at least one optimizer exists. We still use the first optimizer here. Or, do you think that `fit_mmrm` can obtain multiple "optimizer" in the same time and return multiple fits? but that can lead to down stream changes

Reply:
Ok then maybe add comment or in doc to describe that first optimizer is used- and yeah assertion sounds good